src/components/TableBulkSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react';
```

Comment:
unit test would be a nice add. Something like (untested):

```
it('should select all items on the current page', () => {
  const onSelectMock = jest.fn();
  const selectedIds = ['id1'];
  const pageDataIds = ['id2', 'id3'];
  const dataIds = ['id1', 'id2', 'id3', 'id4'];

  const { getByText } = render(
    <TableBulkSelect
      selectedIds={selectedIds}
      pageDataIds={pageDataIds}
      dataIds={dataIds}
      onSelect={onSelectMock}
    />
  );

  fireEvent.click(getByText('Select Page')); // Simulate selecting the page
  expect(onSelectMock).toHaveBeenCalledWith(['id1', 'id2', 'id3']);
});
```

Reply:
Added unit tests.